build/fuses/fuses.json

```diff
@@ -0,0 +1,6 @@
+{
+  "_comment": "Modifying the fuse schema in any way should result in the _version prop being incremented.  NEVER REMOVE A FUSE, instead mark it as removed with '-1'",
```

Comment:
should the version be incremented even when e.g. adding a fuse?

Reply:
@nornagon No, just when modifying the scheme (i.e. the 0 vs 1 vs r bit)

Reply:
then this comment should probably read "modifying an existing fuse ..." or something like that.